bin/deploy

```diff
@@ -7,15 +7,15 @@ ENVIRONMENT=$1
 
 deploy_branch() {
 # Set the deployment host, this will add the prefix of the branch name e.g el-257-deploy-with-circleci or just main
-  RELEASE_HOST="$BRANCH_RELEASE_NAME-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk"
+  RELEASE_HOST="$BRANCH_RELEASE_NAME-check-client-qualifies-legal-aid-$ENVIRONMENT.cloud-platform.service.justice.gov.uk"
```

Comment:
Ooh, could we put a BRANCH_HOST_SUFFIX in the two context we have, and in the `laa-estimate-eligibility-uat` context it could be `check-client-qualifies-legal-aid-uat`, but in the `laa-check-client-qualifies-uat` one it could be `ccq-uat` maybe? THat way this line could be:

```bash
RELEASE_HOST="$BRANCH_RELEASE_NAME-$BRANCH_HOST_SUFFIX.cloud-platform.service.justice.gov.uk"
```

And we'd end up with more concise UAT URLs?

Reply:
I am not even sure this $ENVIRONMENT is required, this script always deploys to UAT so it should never have a production or staging.

Reply:
Yes that is a nice idea! We should try and do that. I can take a look!